examples/computer_vision/cifar10_pytorch/model_def.py

```diff
@@ -104,17 +108,19 @@ def build_training_data_loader(self) -> Any:
         transform = transforms.Compose(
             [transforms.ToTensor(), transforms.Normalize((0.5, 0.5, 0.5), (0.5, 0.5, 0.5))]
         )
-        trainset = torchvision.datasets.CIFAR10(
-            root=self.download_directory, train=True, download=True, transform=transform
-        )
+        with filelock.FileLock(os.path.join(self.download_directory, "lock")):
```

Comment:
Can you put a comment above this line that says:
```python
# Use a file lock so only one worker on each node does the download.
```

I don't think it's obvious, especially to non-dtrain-aware users, why we would use a filelock here.